apps/web/src/pages/Profile/AccountSettings/NotificationSettings.tsx

```diff
@@ -0,0 +1,310 @@
+      ),
+    },
+    {
+      value: "inApp",
+      label: (
+        <span data-h2-visually-hidden="base(invisible)">
+          {intl.formatMessage(commonMessages.inApp)}
+        </span>
+      ),
+    },
+  ];
+
+  if (disabled) {
+    return (
+      <div
+        data-h2-display="base(grid)"
+        data-h2-grid-template-columns="base(4fr 1fr)"
+        data-h2-align-items="base(center)"
+        data-h2-margin-bottom="base(x1)"
+      >
+        <div>
+          <p
+            data-h2-font-weight="base(bold)"
+            data-h2-padding-bottom="base(x.5)"
+          >
+            {legend}
+          </p>
+          <p data-h2-font-size="base(caption)">{subtitle}</p>
+        </div>
+        <div
+          data-h2-display="base(flex)"
+          data-h2-justify-content="base(space-between)"
+        >
+          {notificationOptions.map(({ value }) => {
+            const key = `${id}-${value}`;
+            return (
+              <span
+                key={key}
+                data-h2-padding="base(x.25 x.75 x.25 x.5)"
+                data-h2-color="base(gray)"
+              >
+                <CheckIcon
+                  aria-label={
+                    value === "email"
+                      ? intl.formatMessage(commonMessages.email)
+                      : intl.formatMessage(commonMessages.inApp)
+                  }
+                  aria-hidden="false"
+                  data-h2-height="base(x1)"
+                  data-h2-width="base(x1)"
+                  data-h2-border="base(thin solid black.light)"
+                  data-h2-radius="base(input)"
+                  data-h2-vertical-align="base(middle)"
+                />
+              </span>
+            );
+          })}
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <Field.Wrapper data-h2-margin-bottom="base(x1)">
+      <Field.Fieldset id={id}>
+        <Field.Legend
+          required={false}
+          data-h2-visually-hidden="base(invisible)"
+        >
+          {legend}
+        </Field.Legend>
+        <div
+          data-h2-display="base(grid)"
+          data-h2-grid-template-columns="base(4fr 1fr)"
+          data-h2-align-items="base(center)"
+        >
+          <div>
+            <p
+              data-h2-font-weight="base(bold)"
+              data-h2-padding-bottom="base(x.5)"
+            >
+              {legend}
+            </p>
+            <p data-h2-font-size="base(caption)">{subtitle}</p>
+          </div>
+          <div
+            data-h2-display="base(flex)"
+            data-h2-justify-content="base(space-between)"
+          >
+            {notificationOptions.map(({ value, label }) => {
+              const checkboxId = `${id}-${value}`;
+              return (
+                <Checkbox
+                  key={checkboxId}
+                  id={checkboxId}
+                  name={name}
+                  label={label}
+                  value={value}
+                  inCheckList
+                />
```

Comment:
Is it possible to use `CheckButton` here instead? :thinking: The `Checkbox` has some padding and gap applied that causes this to be slightly off center. Also, the `CheckButton` label is always hidden so that won't need to be done explicitly.

Reply:
Ya I having trouble trying figure out a way to fix the slight gap without having to add a prop just for this one case. I can't use the `Checkbutton`, since it isn't a form field though. It's just a `button` disguised as a checkbox.

Reply:
Oh right :confounded: So it seems like we forward any props from `Checklist` to the `input`.

You could try and cancel out the spacing with something like: `data-h2-margin="base(0 -x.25 0 0)"` :thinking: